core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -378,7 +378,7 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                    style="background-color: <[collection.thumbnail_bg_color]>;">
               </div>
               <md-card class="oppia-activity-summary-tile top-card">
-                <a ng-href="<[getCollectionUrl(collection.id)]>">
+                <a ng-href="<[getCollectionUrl(collection.id)]>" style="color: black;">
```

Comment:
I think that definitely seems backwards. It would be better to change the color to green at the specific places where green is needed, instead of changing it to green universally and doing a series of hacks to change it back.